docs/Course Wiki/ECE Course Offerings/ECE391.md

```diff
@@ -43,9 +60,23 @@ In MP2, you will implement much of the software for a game with visual graphics.
 
 In MP3, you will work in a group of 4 to create an operating system from scratch. This operating system will use paging, a virtual memory system. It will also implement a file system and multiple device drivers for things like a keyboard and real-time clock. The most important part of this MP is implementing system calls, to allow user code to be executed and interact with the kernel, and using your device drivers for input and output. Finally, students implement scheduling to allow for multiple processes to run concurrently.
 
+## Course Structure (Fall 2024)
```

Comment:
Under this make two smaller headers of header 3 and make 2 sections for exams and mps

Reply:
add more info on the content of exams (not actual questions but like topics on each one)